src/SixLabors.Fonts/GlyphMetric.cs

```diff
@@ -11,39 +13,36 @@ namespace SixLabors.Fonts
         /// <summary>
         /// Initializes a new instance of the <see cref="GlyphMetric"/> struct.
         /// </summary>
-        /// <param name="codePoint">Unicode codepoint of the character.</param>
+        /// <param name="codePoint">Unicode codepoint for the glyph.</param>
```

Comment:
I don’t know how helpful this is but instead of codepoint <see cref=“CodePoint” /> might give a subtle addition to show it points to that class. I guess that should be clear from the parameter type though